src/model/Spec.ts

```diff
@@ -199,22 +211,34 @@ export default class Spec {
 
       const schemaValue = isDirectRef ? responseFields : responseFields.content['application/json'].schema;
 
-      if (!this.isRef(schemaValue)) return [response, responseFields];
+      const buildOneOfSchema = (schemaValue: Value): Value => {
```

Comment:
Could we extract this to a private class member as it is repeated twice?

Reply:
Done ✅